examples/ensemble/plot_feature_transformation.py

```diff
@@ -18,6 +18,9 @@
 The resulting transformer has then learned a supervised, sparse,
 high-dimensional categorical embedding of the data.
 
+For a more extended example see
```

Comment:
This change does not make sense: you are linking the example in the example itself. You might want to review the instruction of the original issue.